wherehows-backend/app/actors/EtlJobActor.java

```diff
@@ -113,11 +117,11 @@ public void onReceive(Object message)
         EtlJobDao.endRun(msg.getWhEtlExecId(), EtlJobStatus.SUCCEEDED, "Job succeed!");
         Logger.info("ETL job {} finished", msg.toDebugString());
 
-        if (msg.getEtlJobName().affectDataset()) {
+        if (props.getProperty(Constant.REBUILD_TREE_DATASET) != null) {
```

Comment:
The tree_builder job is not used internally anymore. We can leave it here for now..